cloud_info/providers/openstack.py

```diff
@@ -269,18 +272,20 @@ def get_templates(self):
         flavor_id_attr = 'name' if self.legacy_occi_os else 'id'
         URI = 'http://schemas.openstack.org/template/'
         for flavor in self.nova.flavors.list(detailed=True):
```

Comment:
Check: https://developer.openstack.org/api-ref/compute/#list-flavors-with-details

is_public (Optional) | query | boolean | Filters the flavor list by only public flavors. By default nonadmin users only see public flavors, and admin users can see additional non public flavors.

Should this call be adjusted to get flavors according to the select_flavors option? info-provider should be callable by non-admin users but the API docs are not very clear to me 😕

Reply:
I don't know in fact, it may complexify the way we build the URL, but simplify the filtering.
As I was unsure I kept the implemented way.
This public/non public usage is not really clear for me, as here it seems that by default people are only able to see public image, so I'm wondering what the reasoning is...